### PR TITLE
Implement workers' pick up & mark as complete functionalities for draft print jobs

### PR DESCRIPTION
**Pull request by raisfathin**
Should be able to reuse most of these later for final print jobs, which will be done in a later PR.

Multiple worker feature is not implemented yet. There is still only one queue (thus one worker).

### CODE REVIEW

File: print_job_queue/templates/draft_queue.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	      action="{% url 'draft_job_mark_completion' job_id=job.id %}">
+	  {% csrf_token %}
+	  <input type="hidden" name="worker_name" value="JohnWorker" />
+	  <button>Mark as complete</button>
+	</form>
+      </li>
+      {% endfor %}
+    </ul>
+
+    <h2>Pending</h2>
```
Comment (fushar) [inline print_job_queue/templates/draft_queue.html:33]: Is it possible to show the pending job's created time (== the time the draft translation was requested to be printed)?

File: print_job_queue/templates/draft_queue.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	      action="{% url 'draft_job_pick_up' job_id=job.id %}">
+	  {% csrf_token %}
+	  <input type="hidden" name="worker_name" value="JohnWorker" />
+	  <button>Pick up job</button>
+	</form>
+      </li>
+      {% endfor %}
+    </ul>
+
+    <h2>Completed</h2>
```
Comment (fushar) [inline print_job_queue/templates/draft_queue.html:48]: Is it possible to show the completed job's updated time (== the time the draft translation was printed and delivered)?

**Review by fushar — APPROVED** [review]

File: print_job_queue/templates/draft_queue.html (hunk 1 above)
Comment (raisfathin) [reply print_job_queue/templates/draft_queue.html:33]: Will be done in later PR as discussed offline.

File: print_job_queue/templates/draft_queue.html (hunk 2 above)
Comment (raisfathin) [reply print_job_queue/templates/draft_queue.html:48]: Will be done in later PR as discussed offline.

**Review by Si-plus-plus — APPROVED** [review]

**Pull request merged**